.travis.yml

```diff
@@ -0,0 +1,19 @@
+os: 
+  - linux
+  - windows
+
+language: node_js
+node_js: 
+  - "10"
+
+jobs:
+  include:
+    - stage: "WATT"
```

Comment:
Maybe WATT-WINDOWS would be more consistent with the other stage names? 
Apart from that let's be consistent in keys order (eg stage, script, os).

Reply:
AFAIR WATT works only in Linux server environment. @lslachciak Am I right? 
I agree with the second part of your comment.

Reply:
Currently Open Source WATT CI check is only enabled for Ubuntu. There were some problems with CI run on Mac, so I assume also that WATT is not supported on Mac now.